### ISSUE TITLE
testthat warnings on raster object formation

### ISSUE THREAD

**Issue by jsta** [jsta/ipdw#5]
Testing blocks that call the raster::raster() function give the following warnings:

> 1. raster should not given warning (@test-raster.R#37) - no non-missing arguments to min; returning Inf
> 2. raster should not given warning (@test-raster.R#37) - no non-missing arguments to max; returning -Inf
### Minimum Reproducible Example

<details>
<summary>

 minimum rep ex. </summary>

``` r
test_that("raster should not given warning", {
    skip_on_cran()
    skip_on_travis()
    skip_on_appveyor()

    m <- matrix(NA, 10, 10)
    foobar <- raster::raster(m, xmn = 0, xmx = 10, ymn = 0, ymx = 10)
    expect_s4_class(foobar, "RasterLayer")
})
```

</details>
The error goes away if I use a non-NA value in my m object but I think that completely NA rasters should still be valid.

**Comment by jsta** [comment]
See, https://stackoverflow.com/questions/24282550/no-non-missing-arguments-warning-when-using-min-or-max-in-reshape2. Is there a min of a length 1 vector being computed?

**Comment by jsta** [comment]
Solved by, f91103ccfe5b4d4e7accfb6d6630f99b4795d0f6
